WordPress/gutenberg
# Fix top toolbar for the customizer widgets sidebar control

## What?


Fixes the top toolbar in the widgets customizer because it was broken by #53526 

## Why?


It's a bug.

## How?


1. Undo the size adjustment via an `important` css line. 
2. Hide the collapse button since there is nothing to reveal

## Testing Instructions





0. Activate a classic theme
1. Visit the customizer
2. Go to widgets
3. Add some widgets
4. Make sure that top toolbar is active in the dot menu next to the inserter
5. Select a widget or other block
6. Top toolbar should be side to side of the customizer sidebar and scrollable
7. No collaose button (carrot icon) should show

### Testing Instructions for Keyboard


## Screenshots or screencast 

**Before**

![Screenshot 2023-08-31 at 13 31 05](https://github.com/WordPress/gutenberg/assets/107534/19ac603f-ba64-4f60-997d-ec5ae752f779)

**After**

<img width="1023" alt="Screenshot 2023-09-07 at 17 44 23" src="https://github.com/WordPress/gutenberg/assets/107534/b8b7455a-2b39-493a-9390-40853458ce42">
